multimodal/src/autogluon/multimodal/utils/config.py

```diff
@@ -362,9 +362,19 @@ def save_pretrained_model_configs(
         model = model.model
     for per_model in model:
         if per_model.prefix.lower().startswith(HF_MODELS):
-            per_model.config.save_pretrained(os.path.join(path, per_model.prefix))
-            model_config = getattr(config.model, per_model.prefix)
-            model_config.checkpoint_name = os.path.join("local://", per_model.prefix)
+            # Skip if config is None
+            if not hasattr(per_model, 'config') or per_model.config is None:
+                logger.warning(f"Skipping saving config for {per_model.prefix} as its config is None")
+                continue
+
+            try:
+                per_model.config.save_pretrained(os.path.join(path, per_model.prefix))
+                model_config = getattr(config.model, per_model.prefix)
+                model_config.checkpoint_name = os.path.join("local://", per_model.prefix)
+            except Exception as e:
```

Comment:
would it be better to error out instead of capturing it quietly?

Reply:
It's probably fine skipping the model if it failed to save here.

Reply:
I have reverted this change after figuring out the root cause is from latest transformers release